api/src/main/java/io/opentelemetry/trace/Span.java

```diff
@@ -23,6 +23,26 @@
 @ThreadSafe
 public interface Span {
 
+  /**
+   * Returns an invalid {@link Span}. An invalid {@link Span} is used when tracing is disabled,
+   * usually because there is no OpenTelemetry SDK installed.
+   */
+  static Span getInvalid() {
+    return PropagatedSpan.INVALID;
+  }
+
+  /**
+   * Returns a non-recording {@link Span} that holds the provided {@link SpanContext} but has no
+   * functionality. It will not be exported and all tracing operations are no-op, but it can be used
+   * to propagate a valid {@link SpanContext} downstream.
+   */
+  static Span getPropagated(SpanContext spanContext) {
```

Comment:
s/getPropagated/toPropagated?

Reply:
I think this should be called `Span.wrap(SpanContext)`. Then we are also safe against renamings of propagated/noop/etc

Reply:
I don't think we are safe against behavior change, this needs to always propagate the SC. But +1 for the name